git-keeper-client/gkeepclient/create_config.py

```diff
@@ -0,0 +1,69 @@
+# Copyright 2017 Thuy Dinh
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program.  If not, see <http://www.gnu.org/licenses/>.
+
+"""
+Provides the function for creating a new client.cfg file.
+"""
+
+from os.path import isfile, expanduser, isdir, join
```

Comment:
For functions from `os` the rest of the codebase does not import them by name. Just `import os` with a blank line after it to separate system imports from local imports. Then use `os.path.isfile()`, etc. This makes it obvious when we are using built-in Python functions vs. internal git-keeper functions.